index.rst

```diff
@@ -50,7 +50,696 @@
+| physical_filter_id   | uint64 | PRIMARY KEY                                      |
++----------------------+--------+--------------------------------------------------+
+| name                 | str    | NOT NULL                                         |
++----------------------+--------+--------------------------------------------------+
+| camera_id            | uint64 | NOT NULL REFERENCES Camera (camera_id)           |
++----------------------+--------+--------------------------------------------------+
+| abstract_filter_id   | uint64 | REFERENCES AbstractFilter (abstract_filter_id)   |
++----------------------+--------+--------------------------------------------------+
+| UNIQUE (name, camera_id)                                                         |
++----------------------------------------------------------------------------------+
+
+Entries in the PhysicalFilter table represent the bandpass filters that can be
+associated with a particular visit.  These are different from AbstractFilters,
+which are used to label Datasets that aggregate data from multiple Visits.
+Having these two different DataUnits for filters is necessary to make it
+possible to combine data from Visits taken with different filters.  A
+PhysicalFilter may or may not be associated with a particular AbstractFilter.
+AbstractFilter is the only DataUnit not associated with either a Camera or a
+SkyMap.
+
++----------------------+--------+-----------------------------------------+
+| *PhysicalSensor*                                                        |
++======================+========+=========================================+
+| physical_sensor_id   | uint64 | PRIMARY KEY                             |
++----------------------+--------+-----------------------------------------+
+| number               | uint16 |                                         |
++----------------------+--------+-----------------------------------------+
+| name                 | str    | NOT NULL                                |
++----------------------+--------+-----------------------------------------+
+| camera_id            | uint64 | NOT NULL REFERENCES Camera (camera_id)  |
++----------------------+--------+-----------------------------------------+
+| group                | str    |                                         |
++----------------------+--------+-----------------------------------------+
+| purpose              | str    | NOT NULL                                |
++----------------------+--------+-----------------------------------------+
+| UNIQUE (number, camera_id)                                              |
++-------------------------------------------------------------------------+
+| UNIQUE (name, camera_id)                                                |
++-------------------------------------------------------------------------+
+
+PhysicalSensors actually represent the "slot" for a sensor in a camera,
```

Comment:
I worry that would leave many people confused about what you put in the slot.

I like PhysicalSensor because it's a very clear concept and the name is perfectly appropriate in almost every context (it's only when you think about replacing sensors that it isn't, and it's quite conceivable that will never happen for LSST).